daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/BlindingInfo.scala

```diff
@@ -13,6 +13,9 @@ import com.daml.lf.value.Value.ContractId
   * "Disclosure" tells us which transaction nodes to communicate to which
   * parties.
   * See https://docs.daml.com/concepts/ledger-model/ledger-privacy.html
+  * Note that rollback nodes are a bit special here: Even if the node
+  * itself is not disclosed, a party will be disclosed if a node was below a
+  * rollback node. That information is not included in blinding info at this point.
```

Comment:
We might want to revisit this later but this is enough for the scenario service and sandbox classic and I’d rather start simple and add something later if we do end up needing it.

Reply:
I agree, but I would have added a  `// TODO https://github.com/digital-asset/daml/issues/8020` with this comment inside the code.